cognite/client/data_classes/filters.py

```diff
@@ -259,10 +259,17 @@ class And(CompoundFilter):
 
     Example:
 
-        Combine an In and an Equals filter::
+        Combine an In and an Equals filter:
+
+        A filter using a tuple property reference
 
             >>> from cognite.client.data_classes.filters import And, Equals, In
-            >>> filter = And(Equals(("some", "property"), 42), In(("another", "property"), ["a", "b", "c"]))
+            >>> filter = And(Equals(('space', 'view_external_id/view_version', 'some_property'), 42), In(('space', 'view_external_id/view_version', 'another_property'), ["a", "b", "c"]))
```

Comment:
I think readability would benefit greatly from splitting this into multiline

Reply:
Also, you are overwriting a built-in (`filter`), I would use a different name (lots of occurrences)